src/modules/tokens/views/Dashboard.tsx

```diff
@@ -88,7 +88,7 @@ export const Dashboard: () => JSX.Element = () => {
       fetchWithCredentials(
         `/api/v1/namespaces/${namespace}/tokens/accounts?limit=${
           MAX_ACCOUNTS + 1
```

Comment:
I think "limit" can now be 1 and MAX_ACCOUNTS can be eliminated

Reply:
good catch 🙏🏻